docs/references/making_release.md

```diff
@@ -1,16 +1,16 @@
 ## How to make a new release on PyPI and conda-forge
 
 ### PyPI
-1. iterate the version number. Our version numbering approach is:
+1. update the changelog to put all the unreleased changes under the new version (leaving the unreleased section empty).
+2. Make a PR for the version change and get it accepted & merged. Only make a PyPI distribution from a clean master branch (no local changes).
+3. iterate the version number as a new tag. Our version numbering approach is:
     1. The first number is for very large changes which might happen every couple of years (e.g. 1.0).
```

Comment:
As per https://semver.org : the major version number is reserved for large _backward incompatible_ changes (i.e. API-breaking). Maybe we should clarify this.

Reply:
We do mention something like that in the base level readme :thinking:

Reply:
We explicitly do NOT follow semantic versioning.

Reply:
From the readme:

We use a `generation.major.minor` version number format. We use the `generation`
number for very significant improvements or major rewrites, the `major` number
to indicate substantial package changes (intended to be released every 3-4 months)
and the `minor` number to release smaller incremental updates (intended to be
released approximately monthly and which usually do not include breaking API
changes). We do our best to provide a significant period (usually 2 major
generations) of deprecation warnings for all breaking changes to the API.
We track all changes in our [changelog](https://github.com/RadioAstronomySoftwareGroup/pyuvdata/blob/master/CHANGELOG.md).

Reply:
Hmmm. I guess this is a conversation for another place, but... why? Semantic versioning has a lot of benefits.

Reply:
We talked about it in detail and felt that it did not meet our needs. Other major science packages (including astropy) do not use it. We can chat about it again in the telecon next week if we have time.

Reply:
We try to follow something like the spirit of semantic versioning, but the majority of our PRs add keywords or other functionality, which under strict semantic versioning would require that the second (minor under semantic, major under our definition) version number get updated in most PRs which seemed silly. There are a large number of packages that do not follow strict semantic versioning for the same reason.

Reply:
So I think the biggest plus of semantic versioning is the knowledge that a specific number represents breaking changes, which I believe is still kept in the pyuvdata system (it's the second number, not the first, but that's OK).